src/main/Organization/OrganizationController.java

```diff
@@ -335,9 +338,10 @@ public Handler enrollOrganization(SecurityUtils securityUtils) {
           ctx.json(OrgEnrollmentStatus.PASS_HASH_FAILURE.toJSON().toString());
           return;
         }
-
+        logger.info(user.getUsername() + " " + user.getFirstName());
         logger.info("Setting password and inserting user and org into Mongo");
         user.setPassword(passwordHash);
+        encryptionController.encryptUser(user, username);
```

Comment:
This method does not maintain any internal state, and can be static. We should therefore remove the EncryptionController private field from OrganizationController.

Reply:
See above change on EncryptionController